docs/appendices/release-notes/5.4.4.rst

```diff
@@ -57,6 +57,9 @@ Fixes
 
   returning the value for ``o['a']`` instead of a ``null``.
 
+- Fixed an issue that caused ``UNION`` to throw ``SQLParseExceptions`` instead
+  of ``AmbiguousColumnExceptions``.
```

Comment:
I would expand this a bit more:
```suggestion
  of ``AmbiguousColumnExceptions``, when 2 or more columns are assigned the same name. e.g.::
```
and then the example query.

Reply:
Thanks for reviewing, will update.